HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1998,6 +2001,8 @@ def self.process_heat_pump_adjustment(hvac_sizing_values, weather, hvac, totalCa
         hvac_sizing_values.Cool_Airflow = cfm_per_btuh * hvac_sizing_values.Cool_Capacity
       elsif hvac.HeatType == HPXML::HVACTypeHeatPumpMiniSplit
         hvac_sizing_values.Cool_Airflow = hvac.RatedCFMperTonCooling[-1] * hvac.CapacityRatioCooling[-1] * UnitConversions.convert(hvac_sizing_values.Cool_Capacity, 'Btu/hr', 'ton')
+      elsif hvac.HeatType == HPXML::HVACTypeHeatPumpPTHP
+        hvac_sizing_values.Cool_Airflow = hvac.RatedCFMperTonCooling[hvac.SizingSpeed] * UnitConversions.convert(hvac_sizing_values.Cool_Capacity, 'Btu/hr', 'ton')
```

Comment:
Hmm. In comparing this to the MSHP code right above, A) why use `hvac.SizingSpeed` instead of `-1` and B) why not use `hvac.CapacityRatioCooling`?

Reply:
I was using the same equation as PTAC for PTHP, but for now these two are equivalent in results since we only model single speed systems. Changed to be the same as MSHP for simplicity.